runtime/tests/tests.rs

```diff
@@ -13,21 +13,21 @@
 *     https://centrality.ai/licenses/lgplv3.txt
 */
 
-use cennznet_primitives::types::{AccountId, Balance, FeeExchange, FeeExchangeV1};
+use cennznet_primitives::types::{AccountId, Balance, FeeExchange, FeeExchangeV1, AssetId};
 use cennznet_runtime::{
 	constants::{asset::*, currency::*},
 	Babe, Call, CennzxSpot, CheckedExtrinsic, ContractTransactionBaseFee, EpochDuration, Event, Executive,
 	GenericAsset, Header, Origin, Runtime, Session, SessionsPerEra, Staking, System, Timestamp, TransactionBaseFee,
-	TransactionByteFee, TransactionPayment, UncheckedExtrinsic,
+    TransactionByteFee, TransactionPayment, UncheckedExtrinsic, ScaleDownFactor,
```

Comment:
Did you run `cargo fmt`?

Reply:
Yes, and I'm confused. Is a tab 4 spaces or a tab?

Reply:
a tab char

Reply:
We need to add .rustfmt.toml to the project to specify our own formatting rather than the defaults. I'm afraid the defaults might change with different versions of rustfmt.

Reply:
Okay we actually have this file but somehow not working for me

Reply:
Will format the files again

Reply:
Done. The problem was with CLION doing its own formatting.